RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -808,7 +815,7 @@ class DeepTauId : public deep_tau::DeepTauBase {
         for(size_t n = 0; n < objects.size(); ++n) {
             const auto& obj = objects.at(n);
             const double deta = obj.polarP4().eta() - tau.polarP4().eta();
-            const double dphi = ROOT::Math::VectorUtil::DeltaPhi(tau.polarP4(), obj.polarP4());
+            const double dphi = reco::deltaPhi(tau.polarP4().phi(), obj.polarP4().phi());
```

Comment:
I think that one can use `reco::deltaPhi(tau.polarP4(), obj.polarP4())`, i.e. not call `phi()` method of four-vector.

Reply:
OK, thanks.